### ISSUE TITLE
insert `thead` into table

### ISSUE THREAD

**Issue by luman-web** [tangbc/vue-virtual-scroll-list#61]
Hi. First of all I want to thank you for this module, it works really well. 
I'm facing with little problem. The code below is rendering table. The table has tbody tag, but also i'd like to have thead tag in the table. Please advise how can I insert thead with its th in the table, if it's possible of course. Thank you.

&lt;virtual-list
  :size=&quot;20&quot;
  :remain=&quot;10&quot;
  rtag=&quot;table&quot;
  wtag=&quot;tbody&quot;
  &gt;
  &lt;tr
     v-for=&quot;(row, ir) in sheet&quot;
     :key=&quot;ir&quot;
     &gt;
    &lt;td&gt;{{ +ir + 1 }}&lt;/td&gt;
    &lt;td
      v-for=&quot;(cell, ic) of row&quot;
      :key=&quot;'c' + (ic + 1) + 'r' + (ir + 1)&quot;
      v-html=&quot;cellDisplayValue(ir + 1, ic + 1)&quot;
      &gt;
    &lt;/td&gt;
  &lt;/tr&gt;
&lt;/virtual-list&gt;

**Comment by tangbc** [comment]
In theory, it may be solved by `<slots />`.

Let me try to work out ……

**Comment by Bernardmyburgh5** [comment]
Any progress with table elements integration? any way we could implement a work around for the time being?

**Comment by Soletiq** [comment]
+1

**Comment by udo-walter** [comment]
Any news on this?

**Comment by tangbc** [comment]
It should be recently

**Comment by luman-web** [comment]
> It should be recently

could you provide a syntax how to insert table heads alongside with a loop that renders <tr> tags, please?

**Comment by Mash19** [comment]
I think you can work this out with the existing component. Just do the Table and THead yourself and have the component render the tbody.

```html
    <table>
        <thead>
            <tr>
                  <th>Loop another array for each column header</th>
           </tr>
       </thead>
       <virtual-list
             :size="20"
             :remain="10"
            rtag="tbody"
           wtag="tr"
         >
             <tr
                 v-for="(row, ir) in sheet"
                :key="ir"
               >
                  <td>{{ +ir + 1 }}</td>
                  <td
                      v-for="(cell, ic) of row"
                      :key="'c' + (ic + 1) + 'r' + (ir + 1)"
                      v-html="cellDisplayValue(ir + 1, ic + 1)"
                  >
                 </td>
              </tr>
         </virtual-list>
   </table>

**Comment by W65138497** [comment]
How to achieve the replacement of product components into table elements

**Comment by xxbld** [comment]
https://github.com/tangbc/vue-virtual-scroll-list/pull/139

**Comment by tangbc** [comment]
Released version 1.4.3 thanks @xxbld
